clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -73,6 +73,12 @@ def NullAttr : CIR_Attr<"Null", "null", [TypedAttrInterface]> {
 
   let parameters = (ins AttributeSelfTypeParameter<"">:$type);
 
+  let builders = [
+    AttrBuilderWithInferredContext<(ins "Type":$type), [{
+      return $_get(type.getContext(), type);
+    }]>
+  ];
+
```

Comment:
Should we skip default builders here?

Reply:
Not possible AFAIK. What happens is that the tablegen-generated parser actually uses the default builder, so skipping it breaks the build.